src/experiments/data/index.js

```diff
@@ -0,0 +1,33 @@
+const { applyFilters, doAction } = wp.hooks;
+
+/**
+ * Return formatted object with defined sidebar tests coming from the backend.
+ *
+ * @returns {object} Tests object.
+ */
+export const getSidebarTests = () => {
+	const input = window.Altis.Analytics.Experiments.PostABTests || {};
+	const tests = {};
+
+	for ( let id in input ) {
+		const test = input[ id ];
+
+		/* eslint-disable jsdoc/require-jsdoc */
+		tests[ id ] = applyFilters( `altis.experiments.sidebar.test.${ id }`, {
+			id,
+			title: test.label,
+			singleTitle: test.singular_label,
+			component: () => ( <></> ),
```

Comment:
Need to import React to this file unless this can default to `null`, if it's `null` we just wouldn't show the UI . Maybe we can infer a default control type from the variants field type setting?

Reply:
Yeah, I had actually wanted to use the default text input type. Good spot!

Reply:
FYI the rule about having to import React is outdated, see https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html , might need to update our rules.